src/Services/WorkItemImprovementWorkflow.php

```
@@ -0,0 +1,111 @@
+        $scored = $this->storyAiServices
+            ->createQualityScorer()
+            ->scoreWorkItem($item, $qualityBlock);
+
+        if ($scored['score'] !== null) {
+            HLWorkItem::markQualityScoredForOrg(
+                $this->db,
+                $id,
+                $orgId,
+                $scored['score'],
+                $scored['breakdown']
+            );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Models\HLWorkItem' in method 'scoreInitialWorkItem'. (undefined)

(StaticAccess)